site/oauth2-examples-keycloak.md

```diff
@@ -109,10 +109,10 @@ rather than `legacy-token-key`.
 
 For backend applications which uses **Client Credentials flow** you create a **Client** with:
```

Comment:
add a comma after  **Client Credentials flow** 

add the word "can" in between "you" and "create"